scale/recipe/messages/process_recipe_input.py

```diff
@@ -84,6 +97,7 @@ def execute(self):
             Recipe.objects.process_recipe_input(recipe)
 
         # Create message to update the recipe
+        # TODO: switch to new update_recipe message and send forced_nodes
```

Comment:
Did you mean to implement this change or is this for a future issue?

Reply:
Future issue: #1177

Reply:
👌